packages/insomnia-app/app/ui/components/settings/theme.js

```diff
@@ -114,13 +119,68 @@ class Theme extends React.PureComponent<Props, State> {
 
     return rows.map((row, i) => (
       <div key={i} className="themes__row">
-        {row.map(this.renderTheme)}
+        {row.map(theme => this.renderTheme(theme))}
```

Comment:
Can be simplified to the previous implementation 🙂 

```suggestion
        {row.map(this.renderTheme)}
```